### ISSUE TITLE
Oculus OpenXR build positions controllers incorrectly

### ISSUE THREAD

**Issue by msub2** [Igalia/wolvic#26]
## Configuration

Wolvic version: 0.9.3
Wolvic build ID: 9fd07714

Hardware: Quest 2 v37

## Steps to Reproduce

1. Launch app built with OpenXR enabled
2. Observe that controllers are positioned incorrectly

## Current Behavior

Controllers are not positioned where my real life controllers are

## Expected Behavior

They should be positioned correctly

## Possible Solution

It appears that the controllers are being positioned at the `aim` pose instead of the `grip` pose for some reason. I'm unsure if this is an issue with the app itself or with one of the OpenXR SDKs.

## Context

Using the [Oculus OpenXR Mobile SDK v37](https://developer.oculus.com/downloads/package/oculus-openxr-mobile-sdk/) and the [OpenXR-SDK files from master](https://github.com/KhronosGroup/OpenXR-SDK).

```
```

</details>

**Comment by msub2** [comment]
I've found the offending line, it's in OpenXRInputSource.cpp at line 431:
```cpp
CHECK_XRCMD(GetPoseState(mPointerAction, mPointerSpace, localSpace, frameState,  isPoseActive, poseLocation));
```

Changing mPointer to mGrip seems to position the controllers closer to where they should be, however their rotation is now off by 90 degrees.

**Comment by MortimerGoro** [comment]
thanks for reporting this. At some point we want to switch to OpenXR API in Oculus. We have other pending task such as battery level, store entitlement validation, and some extensions

**Comment by svillar** [comment]
I've checked the OpenXR build with the latest code and I do see indeed a small misplacement. I have both controllers together but in VR I see them separated, rotation looks good, and the location is aproximately correct but with some offset in the X axis.

**Comment by svillar** [comment]
This should have been fixed in 0.9.6
